README.md

```diff
@@ -879,8 +879,10 @@ API | Description | Auth | HTTPS | CORS |
 | [Bitly](http://dev.bitly.com/get_started.html) | URL shortener and link management | `OAuth` | Yes | Unknown |
 | [CleanURI](https://cleanuri.com/docs) | URL shortener service | `No` | Yes | Yes |
 | [ClickMeter](https://support.clickmeter.com/hc/en-us/categories/201474986) | Monitor, compare and optimize your marketing links | `apiKey` | Yes | Unknown |
+| [LiteLink](https://litelink.ml/) | Simple URL Shortener | `No` | Yes | Yes |
```

Comment:
It looks that `https://litelink.ml/` doesn't work anymore. Can you please update it?

Reply:
Worked for me, example: https://www.litelink.ml/MjQ=

Reply:
Yup, works for me too. @marekdano, could you pls re-check and discard the pull request if it fails for you again?